environments/kubernetes/manifests/jaeger-deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app.kubernetes.io/name: jaeger-all-in-one
+  name: jaeger-all-in-one
+  namespace: observatorium
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: jaeger-all-in-one
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: jaeger-all-in-one
+    spec:
+      containers:
+      - args:
+        - --badger.directory-key=/var/jaeger/store/keys
```

Comment:
``--badger.ephemeral=false`` is required, otherwise these parameters are ignored. What sort of TTL has been discussed? Default is 3 days.

Reply:
3 days should be more than necessary. I would even say 24h is totally fine. Happy to try 3d first though.

Reply:
Cool, I thought the flag would be adjusted to `false` when the key/values options were set.